apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -93,7 +96,8 @@ export class ModalPage {
     if (otp.replace(' ', '').length !== 6) {
       otp = email.getOtpCodeFromBody(emailBody)
     }
-    await this.enterOTP(otp)
+    // Remove empy space in OTP code 111 111
+    await this.enterOTP(otp.replace(' ', ''))
```

Comment:
maybe the `replace` could be in `getOtpCodeFromBody` to prevent repeating

Reply:
done